docs/tutorial.rst

```diff
@@ -19,20 +19,19 @@ command as in the following example.
    $ pgcopydb clone
 
 Note that the options ``--source`` and ``--target`` can also be used to set
-the Postgres connection strings to the databases. Using environment
-variables is particulary useful when using Docker containers.
+the Postgres connection strings to the databases; however, using *environment
+variables* is particulary useful when using Docker containers.
 
 You might also notice here that both the source and target Postgres
-databases must already exist for pgcopydb to operate.
+databases must already exist for ``pgcopydb`` to operate.
 
 Copy Postgres users and extensions
 ----------------------------------
 
-To copy Postgres users a privileged connection to the target database must
-be made, and to include passwords a privileged connection to the source
-database must be made too. It is sometimes wanted to limit these privileged
-connection to the minimum, and then an approach that looks like the
-following may be used:
+To copy Postgres users, a privileged connection to the target database must
+be setup, and to include passwords, a privileged connection to the source
+database must be present as well. If it is required to limit these privileged
+connections to a minimum, then the following approach may be used:
```

Comment:
The rephrasing is overall better, but “must be present as well” reads wrong. The connection must be made, established, setup. Not “present”.

Reply:
Reworded present as setup. I think the rest is fine.